src/app/business/common/components/detail/components/metadata.js

```diff
@@ -32,6 +33,14 @@ const baseDd = css`
     margin-bottom: ${spacingExtraSmall};
 `;
 
+const metadataDt = css`
+    text-transform: initial;
```

Comment:
I'd also have monospace here, and a little left padding (using the spacing variable) so that it's clear that it's under metadata.

Reply:
Yes, I tried two different ways of doing it, which one do you prefer?
<img width="811" alt="Screenshot 2020-07-24 at 10 34 04 AM" src="https://user-images.githubusercontent.com/45589315/88374819-bbf0ce80-cd9a-11ea-8c21-a66b9385f681.png">

<img width="814" alt="Screenshot 2020-07-24 at 10 38 05 AM" src="https://user-images.githubusercontent.com/45589315/88374870-d7f47000-cd9a-11ea-83ad-5fbd7191f3b1.png">

Reply:
The second one seems perfect to me! It makes it clear that data/foo/test are sub-sections of METADATA while not breaking the style at all, very well done!